packit/upstream.py

```diff
@@ -516,7 +517,9 @@ def fix_spec(self, archive: str, version: str, commit: str):
             changelog_entry=msg,
         )
 
-    def _fix_spec_prep(self, version):
+    def _fix_spec_prep(
+        self, archive, archive_root_dir_template="{upstream.pkg_name}-{version}"
```

Comment:
I'm probably blind but I can't see archive_root_dir_template being used anywhere in the method

Reply:
is in  

https://github.com/packit/packit/blob/95d43e1978021bb5d375e46acf0e1f60a73d7eea/packit/upstream.py#L926

https://github.com/packit/packit/blob/95d43e1978021bb5d375e46acf0e1f60a73d7eea/packit/upstream.py#L945-L946

moved it here to keep separate responsibility of methods.
I need to update the method docstring.

Reply:
okay, I just opened this in pycharm and indeed `archive_root_dir_template` is not used within the _fix_spec_prep method, so please remove it from the arguments:
```
Parameter 'archive_root_dir_template' value is not used 
```

plus there is typo in the default, there should be an underscore instead of a dot

Reply:
ok, I understand now, it's just leftover - removed